exercisePPJ_05_VII/ExercisePPJ_05_VII.java

```diff
@@ -0,0 +1,61 @@
+/*
+Dana jest zmienna typu boolean czyPada zainicjowana wartością true, oraz zmienna typu boolean czySwieciSlonce zainicjowaną dowolną wartością.
+Napisz program wykorzystujący instrukcję warunkową if i wyświetlający stany pogody jako:
+• plucha – gdy pada i nie świeci słońce,
+• tęcza – gdy pada i świeci słońce
+• słonecznie - gdy świeci słońce i nie pada,
+• pochmurno - gdy nie świeci słońce i nie pada.
+*/
+
+import java.util.Scanner;
+
+public class ExercisePPJ_05_VII {
+	
+	public static void main(String[] args) {
+	
+	boolean czyPada = true;
+	boolean czySwieciSlonce = true;
+	
+	System.out.println("Weather");
+	System.out.println("Please enter one from dwo numbers: (1) pada / (2) nie pada");
+	
+	Scanner input1 = new Scanner(System.in);
+	int a = input1.nextInt();
+
+	System.out.println("Please enter one form dwo numbers: (3) swieci / (4) nie swieci");
+
+	Scanner input2 = new Scanner(System.in);
+	int b = input2.nextInt();
+	
+	if(a == 1) {
+		czyPada = true;
+	}
+	else {
+		czyPada = false;
+	}
+	
+	if(b == 3) {
+		czySwieciSlonce = true;
+	}
+	else {
+		czySwieciSlonce = false;
+	}
+	
+	if((czyPada == true) && (czySwieciSlonce == false)) {
```

Comment:
if statement takes `boolean` as argument. So, you can check `if(czyPada && czySwieciSlonce)`
Do you understand it?:)

Reply:
@MatDls 
but I compare four times this. if I write 4x if(czyPada && czySwieciSlonce) thats useless. I must check that czyPada or czySwieciSlonce changed